package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@adminjs/common",
+  "version": "7.0.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "license": "MIT",
+  "author": {
+    "email": "rafal.dzieg@gmail.com",
+    "name": "Rafał Dzięgielewski"
+  },
+  "scripts": {
+    "clean": "rimraf utils constants.js index.js index.d.ts",
+    "build": "yarn clean && tsc",
+    "dev": "yarn clean && tsc -w",
+    "test": "mocha ./spec/index.js"
+  },
+  "dependencies": {
+    "@babel/core": "^7.18.2",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.18.2",
+    "@babel/preset-react": "^7.17.12",
+    "@babel/preset-typescript": "^7.17.12",
+    "@babel/register": "^7.17.7",
+    "flat": "^4.1.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.3.1",
+    "@types/flat": "^0.0.28",
+    "@types/mocha": "^9.1.1",
+    "@types/node": "^17.0.38",
+    "@typescript-eslint/eslint-plugin": "^5.27.0",
+    "@typescript-eslint/parser": "^5.27.0",
+    "chai": "^4.3.6",
```

Comment:
why `chai` instead of `jest`?

Reply:
the tests are copied from core repo, I would also prefer to use `jest` but I prioritise frontend separation for now